app/forms/declaration/CommodityMeasure.scala

```diff
@@ -28,30 +28,62 @@ import utils.validators.forms.FieldValidator._
 
 import scala.util.Try
 
-case class CommodityMeasure(supplementaryUnits: Option[String], grossMass: Option[String], netMass: Option[String])
+case class CommodityMeasure(
+  supplementaryUnits: Option[String],
+  supplementaryUnitsNotRequired: Option[Boolean],
```

Comment:
Is there any situation when this equals `Some(false)` ? I'm suggesting, that maybe unwrapped `Boolean` would be enough?

Reply:
Yes, if the user does not tick the supplementaryUnitsNotRequired checbox. I think its is better that this value is optional to reflect that the user has not yet made the yes/no determination on this field.